code/modules/projectiles/projectile/beams.dm

```diff
@@ -84,6 +84,11 @@
 	damage = 0
 	nodamage = TRUE
 
+/obj/projectile/beam/laser/slug
+	name = "laser slug"
+	icon_state = "heavylaser"
+	damage = 30
```

Comment:
imo it should have some AP for being a more focused shot

Reply:
If it had AP it would again be relegated to always being on kill (or in this case, slug) mode. Not that i'm unwilling to change this with time, but i'd like to see how players use it first. Then maybe buff it if it needs it.
Or I can reduce the damage and give AP which would make it basically the same, idk.

I am willing to try it out with AP & less damage.

Reply:
AP and less damage would give it another interesting niche imo.

Reply:
Will give it a shot when I push the damage fall-off to the scatter mode